coalib/results/Result.py

```diff
@@ -15,6 +15,8 @@
                ('severity', RESULT_SEVERITY.reverse.get),
                'confidence',
                'message',
+               'message_base',
+               'message_arguments',
                ('aspect', lambda aspect: type(aspect).__qualname__))
 @generate_ordering('affected_code',
```

Comment:
we shouldsort by message_base and then by messa_arguments, not by message IMO this way we get the categories properly separated